client/src/components/ResultVisualization.vue

```diff
@@ -521,7 +524,16 @@ export default defineComponent({
                 .selectAll('circle')
                 .classed('selected', false)
                 .filter((td) => td === node)
+                //highlight border
                 .classed('selected', true);
+
+            this.$emit('onPatentSelected', { patent: node.patent, index: node.index ?? -1 });
```

Comment:
This line causes the patent preview to be broken after clicking a non-patent node. Maybe it got here by accident during a merge?